CMakeLists.txt

```diff
@@ -128,7 +129,9 @@ set_package_properties(Qt6Network PROPERTIES TYPE REQUIRED)
 set_package_properties(Qt6Svg PROPERTIES TYPE REQUIRED)
 
 find_package(Taglib REQUIRED taglib>=1.12)
+if(BUILD_ALSA)
 find_package(ALSA REQUIRED)
+endif()
```

Comment:
You'll need to add the definition you're using here so it can be found:

`add_definitions(-DBUILD_ALSA_ENABLED)`

Reply:
Ok fixed.